### PR TITLE
chore(shorebird_cli): v1.0.1

### PR DESCRIPTION
**Pull request by felangel**
## Description

- chore(shorebird_cli): v1.0.1

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [X] 🗑️ Chore

### CODE REVIEW

**Review by eseidel — APPROVED** [review]
Looks great!

**Comment by eseidel** [comment]
Hmm, maybe we should link to https://docs.shorebird.dev/status, and I should add https://github.com/shorebirdtech/shorebird/issues/1619 to status since it keeps coming up.

**Comment by felangel** [comment]
> Hmm, maybe we should link to https://docs.shorebird.dev/status, and I should add #1619 to status since it keeps coming up.

Done, lmk if that's what you had in mind.

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -2,6 +2,18 @@
...
+
+- 🚀 Improve link percentage by ~10-15% for all iOS patches
+- 🩹 Fix potentially incorrect behavior in optimized try/catch on iOS patches
+- 🥞 Fix potential crashes when using dwarf stack traces on iOS patches
+- 📝 Include patch instructions after all `release` commands
+- 🔗 Link to the Shorebird Console when a release already exists
+- 🚉 Support `--target-platforms` on android release and patch commands
+- 🍧 Fix `shorebird release android` with camelCase flavor names
+
+ℹ️ See [Shorebird Status](https://docs.shorebird.dev/status) up-to-date status.
```
Comment (eseidel) [inline RELEASE_NOTES.md]:
```suggestion
ℹ️ [Shorebird Status](https://docs.shorebird.dev/status) tracks any known issues.
```

**Comment by eseidel** [comment]
🤷  Thanks!

**Pull request merged**